<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сетевое планирование в разработке СТС</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Параметры проекта</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Директивное время выполнения:</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="sbWorkersCount">
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Число этапов:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSpinBox" name="sbDirTime">
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Число исполнителей / работ:</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QSpinBox" name="sbStageCount">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>70</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTabWidget" name="tw">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabCost">
       <attribute name="title">
        <string>Стоимость</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTableWidget" name="twCost">
          <row>
           <property name="text">
            <string>Исполнитель 1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Исполнитель 2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Исполнитель 2</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Работа 1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Работа 2</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Работа 2</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabTime">
       <attribute name="title">
        <string>Время</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTableWidget" name="twTime">
          <row>
           <property name="text">
            <string>Исполнитель 1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Исполнитель 2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Исполнитель 2</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Работа 1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Работа 2</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Работа 2</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Сетевой график</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Матрица смежности:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableWidget" name="twNet"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>688</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHfccxtn">
    <property name="title">
     <string>Рассчет</string>
    </property>
    <addaction name="aCountAssesments"/>
    <addaction name="aCountCriticalPath"/>
   </widget>
   <addaction name="menuHfccxtn"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="aCountAssesments">
   <property name="text">
    <string>Назначений</string>
   </property>
  </action>
  <action name="aCountCriticalPath">
   <property name="text">
    <string>Критического пути</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
